src/main/java/org/dbflute/intro/app/logic/intro/IntroPhysicalLogic.java

```diff
@@ -21,7 +21,8 @@
 import org.dbflute.util.Srl;
 
 /**
- * The logic for DBFlute Intro physical operation.
+ * The logic for DBFlute Intro physical operation. <br>
+ * Introの全体的な物理情報を扱う。DBFluteクライアント配下などは、それぞれ個別のLogicクラスにて。
```

Comment:
より具体的に言うとこういうことでしょうか？「全体的な」ってどういうことだろうと思いまして 🤔 
```suggestion
 * Introが起動しているディレクトリの情報を扱う。DBFluteクライアント配下などは、それぞれ個別のLogicクラスにて。
```